lib/recommended-rules.js

```diff
@@ -5,56 +5,57 @@
  * definitions, execute "npm run update"
  */
 module.exports = {
-  "ember/alias-model-in-controller": "off",
-  "ember/avoid-leaking-state-in-ember-objects": "error",
-  "ember/avoid-using-needs-in-controllers": "error",
-  "ember/classic-decorator-hooks": "off",
-  "ember/classic-decorator-no-classic-methods": "off",
-  "ember/closure-actions": "error",
-  "ember/computed-property-getters": "off",
-  "ember/jquery-ember-run": "error",
-  "ember/named-functions-in-promises": "off",
-  "ember/new-module-imports": "error",
-  "ember/no-arrow-function-computed-properties": "error",
-  "ember/no-attrs-in-components": "error",
-  "ember/no-attrs-snapshot": "error",
-  "ember/no-capital-letters-in-routes": "error",
-  "ember/no-deeply-nested-dependent-keys-with-each": "error",
-  "ember/no-duplicate-dependent-keys": "error",
-  "ember/no-ember-super-in-es-classes": "error",
-  "ember/no-ember-testing-in-module-scope": "error",
-  "ember/no-empty-attrs": "off",
-  "ember/no-function-prototype-extensions": "error",
-  "ember/no-get": "off",
-  "ember/no-global-jquery": "error",
-  "ember/no-incorrect-calls-with-inline-anonymous-functions": "error",
-  "ember/no-invalid-debug-function-arguments": "error",
-  "ember/no-jquery": "off",
-  "ember/no-new-mixins": "error",
-  "ember/no-observers": "error",
-  "ember/no-old-shims": "error",
-  "ember/no-on-calls-in-components": "error",
-  "ember/no-pause-test": "off",
-  "ember/no-proxies": "off",
-  "ember/no-restricted-resolver-tests": "error",
-  "ember/no-side-effects": "error",
-  "ember/no-test-and-then": "off",
-  "ember/no-test-import-export": "off",
-  "ember/no-unnecessary-index-route": "off",
-  "ember/no-unnecessary-route-path-option": "error",
-  "ember/no-unnecessary-service-injection-argument": "off",
-  "ember/no-volatile-computed-properties": "error",
-  "ember/order-in-components": "off",
-  "ember/order-in-controllers": "off",
-  "ember/order-in-models": "off",
-  "ember/order-in-routes": "off",
-  "ember/require-computed-macros": "off",
-  "ember/require-computed-property-dependencies": "off",
-  "ember/require-return-from-computed": "error",
-  "ember/require-super-in-init": "error",
-  "ember/route-path-style": "off",
-  "ember/routes-segments-snake-case": "error",
-  "ember/use-brace-expansion": "error",
-  "ember/use-ember-data-rfc-395-imports": "off",
-  "ember/use-ember-get-and-set": "off"
-}
\ No newline at end of file
+  'ember/alias-model-in-controller': 'off',
```

Comment:
Looks like Prettier reformatted to be single quotes

Reply:
I'll fix this separately in #547